source/dotnet/Library/AdaptiveCards/ParseContext.cs

```diff
@@ -24,26 +24,36 @@ public enum ContextType { Element, Action };
         // Push the provided state on to our ID stack
         public static void PushElement(string idJsonProperty, AdaptiveInternalID internalId)
         {
+// re-enable when proper fix is checked-in
+#if USEFALLBACK 
```

Comment:
>USEFALLBACK [](start = 4, length = 11)

Where is this being configured again?

Reply:
yes, instead of commenting it out the define; i just didn't add it since we won't be debugging it.; but i will addi it.

---
In reply to: [340390172](https://github.com/microsoft/AdaptiveCards/pull/3546#discussion_r340390172) [](ancestors = 340390172)